nemo/collections/asr/modules/conformer_encoder.py

```diff
@@ -60,6 +64,12 @@ class ConformerEncoder(NeuralModule, StreamingEncoder, Exportable):
             Defaults to 4.
         subsampling_conv_channels (int): the size of the convolutions in the subsampling module
             Defaults to -1 which would set it to d_model.
+        reduction (str, Optional): the method of reduction, choices=['pooling', 'subsampling']. If no value
```

Comment:
What do you mean by not passing any value? do you mean None?

Reply:
If the reduction parameter is not specified in the config then no such parameter is passed while creating an object of this class so the values are by default None.